docs/en/compute/Tensorflow.md

```diff
@@ -14,44 +14,52 @@ This guide describes how to run [Tensorflow](https://www.tensorflow.org/) on top
 ## Overview
 
 Tensorflow enables developers to quickly and easily get started with deep learning. 
-In the [deep learning]({{ '/en/compute/Deep-Learning.html' | relativize_url }}) section, 
-we illustrate the data challenges of deep learning and how Alluxio helps to solve
-the challenges. In this tutorial, we will provide some hands-on examples and tips for running Tensorflow
+The [deep learning]({{ '/en/compute/Deep-Learning.html' | relativize_url }}) section illustrates the data challenges of deep learning 
+and how Alluxio helps to solve those challenges. This tutorial aims to provide some hands-on examples and tips for running Tensorflow
 on top of Alluxio-FUSE.
 
 ## Prerequisites
 
 * Setup Java for Java 8 Update 60 or higher (8u60+), 64-bit.
 * Alluxio has been set up and is running.
+* [Tensorflow](https://www.tensorflow.org/install/pip) installed. 
 
 ## Setting up Alluxio-FUSE
 
-In this section, we will follow the instructions in the
-[FUSE section]({{ '/en/api/FUSE-API.html' | relativize_url }}) to set up FUSE 
-and allow Tensorflow applications to access the data through FUSE.
+Run the following command to install FUSE on Linux:
 
-Create a folder at the root in Alluxio
+```
+$ yum install fuse fuse-devel
+```
+
+On MacOS, download the [osxfuse dmg file](https://github.com/osxfuse/osxfuse/releases/download/osxfuse-3.8.3/osxfuse-3.8.3.dmg) instead and follow the installation instructions.
+
+Create a folder at the root in Alluxio: 
 
 ```bash
 $ ./bin/alluxio fs mkdir /training-data
 ```
 
-Start the Alluxio-FUSE process. Create a folder `/mnt/fuse`, change its
-owner to the current user (`ec2-user` in this example), and modify its permissions to allow read and write.
+Create a folder `/mnt/fuse`, change its owner to the current user (`$(whoami)`), 
+and change its permissions to allow read and write:
 
 ```bash
 $ sudo mkdir -p /mnt/fuse
-$ sudo chown ec2-user:ec2-user /mnt/fuse
-$ chmod 664 /mnt/fuse
+$ sudo chown $(whoami) /mnt/fuse
+$ chmod 755 /mnt/fuse
 ```
 
-Run the Alluxio-FUSE shell to mount Alluxio folder `training-data` to the local folder
-`/mnt/fuse`.
+Run the Alluxio-FUSE shell to mount Alluxio folder `training-data` to the local empty folder
+just created:
 
 ```bash
 $ ./integration/fuse/bin/alluxio-fuse mount /mnt/fuse /training-data
 ```
 
+The above CLI spawns a background user-space java process (`alluxio-fuse`) that mounts the Alluxio path specified at `/training-data` 
+to the local file system on the specified mount point `/mnt/alluxio`. Please refer to [FUSE documentation]({{ '/en/api/FUSE-API.html' | relativize_url }}) 
```

Comment:
Let's refer to it as `POSIX API documentation`

Reply:
Changed, thanks